demo-app/src/main/kotlin/io/getstream/video/android/data/services/stream/StreamService.kt

```diff
@@ -24,23 +24,22 @@ import retrofit2.create
 import retrofit2.http.GET
 import retrofit2.http.Query
 
-interface StreamTokenService {
-
+interface StreamService {
     @GET("api/auth/create-token")
-    suspend fun fetchToken(
+    suspend fun getAuthData(
+        @Query("environment") environment: String,
         @Query("user_id") userId: String?,
-        @Query("api_key") apiKey: String,
-    ): TokenResponse
-}
+    ): GetAuthDataResponse
 
-object StreamVideoNetwork {
+    companion object {
+        private const val BASE_URL = "https://pronto.getstream.io/"
 
-    private val contentType = "application/json".toMediaType()
-    private val retrofit =
-        Retrofit.Builder()
-            .baseUrl("https://stream-calls-dogfood.vercel.app/")
-            .addConverterFactory(Json.asConverterFactory(contentType))
-            .build()
+        private val retrofit =
+            Retrofit.Builder()
+                .baseUrl(BASE_URL)
+                .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
```

Comment:
Default `Json` is too strict for parsing, can we also fix it here (similar like yesterda with login) and use 
```
    private val json = Json {
        ignoreUnknownKeys = true
    }
```

?